Cargo.toml

```diff
@@ -180,6 +180,7 @@ opt-level = 0
 [profile.bench]
 lto = false
 debug = true
+codegen-units = 256 # restore default value for faster compilation
```

Comment:
Is it possible to to remove this line and add check in `[profile.release]` if it is a non-bench build?

Reply:
cause I had to open rust documentation to check the default value